bundles/org.openhab.binding.juicenet/README.md

```diff
@@ -0,0 +1,215 @@
+# JuiceNet Binding
+
+The JuiceNet binding will interface with the cloud portal to get status and manage your JuiceBox EV charger(s).
+In addition to getting the status of various items from the ev-charger, it is also possible to start and stop charging sessions.
+
+## Supported Things
+
+This binding supports the following things:
+
+| thing               | type    | description                  |
+|----------           |-------- |------------------------------ | 
+| JuiceNet Account    | Bridge  | This represents the cloud account to interface with the JuiceNet API.  |
+| JuiceBox EV-Charger | Device  | This interfaces to a specific JuiceBox EV-charger associated with the JuiceNet account. |
+
+This binding should work with multiple JuiceBox EV-chargers associated with the account, however it is currently only tested with a single EV-charger.
+
+## Thing Configuration
+
+The only configuration required is to create a JuiceNet account thing and fill in the appropriate API Token.
+The API token can be found on the Account page at https://home.juice.net/Manage.
+
+Once, the JuiceNet Account thing has been created, all JuiceBox EV-chargers associated with that account will be discovered and added to the inbox.
+
+## Channels
+
+| channel           | type              | read-only | description                  |
+|----------         |--------           |------------------------------|
```

Comment:
This is not correctly formatted and will be wrongly rendered.

Reply:
Thanks - have you been able to get tables to correctly render in the Eclipse Preview?

Reply:
I haven't tried that, sorry. I'm using the viewer in Directory Opus sometimes, as well as the GitHub viewer when changes are pushed.